specs/Queue.spec.js

```diff
@@ -0,0 +1,51 @@
+var queue = require('../src/Queue');
+var assert = require('assert');
+
+describe('Queue', function () {
+    describe('push', function () {
+        it('should return last push arguments', function() {
+            let Queue = new queue(5);
+
+            assert.equal(Queue.length + 3, Queue.push(2,3,4));
+            delete Queue;

```

Comment:
1st delete is not working on variables.
2nd no need to remove object inside tests
3rd no code after assertions
4th assertion and execution should be separated